djangocms_navigation/cms_config.py

```diff
@@ -23,9 +28,54 @@ def configure_app(self, cms_config):
             )
 
 
-class CoreCMSAppConfig(CMSAppConfig):
-    djangocms_navigation_enabled = True
+def copy_menu_content(original_content):
+    """Copy the MenuContent object and deepcopy its menu items."""
+    # Copy root menu item
+    original_root = original_content.root
+    root_fields = {
+        field.name: getattr(original_root, field.name)
+        for field in MenuItem._meta.fields
+        if field.name not in [MenuItem._meta.pk.name, "path", "depth"]
+    }
+    new_root = MenuItem.add_root(**root_fields)
+
+    # Copy MenuContent object
+    content_fields = {
+        field.name: getattr(original_content, field.name)
+        for field in MenuContent._meta.fields
+        if field.name not in [MenuContent._meta.pk.name, "root"]
+    }
+    content_fields["root"] = new_root
+    new_content = MenuContent.objects.create(**content_fields)
+
+    # Copy menu items
+    for item in MenuItem.get_tree(original_root).exclude(pk=original_root.pk):
+        item_fields = {
+            field.name: getattr(item, field.name)
+            for field in MenuItem._meta.fields
+            if field.name not in [MenuItem._meta.pk.name, "path"]
+        }
+        item_fields["path"] = new_root.path + item.path[MenuItem.steplen:]
+        MenuItem.objects.create(**item_fields)
```

Comment:
Can we use bulk_create here? Should improve performance.

Reply:
@chaosk I'm not sure... treebeard does a lot of funky custom db stuff and and django's `bulk_create`  also has some caveats such as not sending out signals or not running the model's save method etc. So I'm a little concerned something might go wrong with `bulk_create`. But if you prefer to go that route anyway, I can.

Reply:
I've checked treebeard's code and `bulk_create` should work in this case. I'd still recommend checking if it doesn't break anything.

Reply:
@chaosk I've got a unit test already where it creates about 7 variously nested menu items. Not really sure what else I can do to test this more thoroughly, is that sufficient do you think?

Reply:
Should be fine then!